### PR TITLE
Add quasar framework docs for generator.

### PR DESCRIPTION
**Pull request by devel-pa**
Hi, 
Added docs for https://github.com/api-platform/client-generator/pull/134
I hope the branch is the right one.

### CODE REVIEW

File: client-generator/quasar.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+[quasar-cli](https://quasar.dev/quasar-cli/installation):
+
+    $ quasar create my-app
+    $ cd my-app
+
+    $ yarn global add @api-platform/client-generator
+
+In the app directory, generate the files for the resource you want:
+
+    $ generate-api-platform-client -g quasar https://demo.api-platform.com src/ --resource foo
```
Comment (dunglas) [inline client-generator/quasar.md]: Could simply be `yarn dlx @api-platform/client-generator -g quasar https://demo.api-platform.com src/ --resource foo`

File: client-generator/quasar.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    # Replace the URL by the entrypoint of your Hydra-enabled API
+    # Omit the resource flag to generate files for all resource types exposed by the API
+
+The code is ready to be executed! Register the generated routes in `src/router/routes.js`
+
+```javascript
+import fooRoutes from '../generated/router/foo';
```
Comment (dunglas) [inline client-generator/quasar.md]: In a next version of the generator, you could use `recast` to automatically add this code (as done in the new Next.js generator). Not related to the docs PR thought.

File: client-generator/quasar.md (hunk 1 above)
Comment (devel-pa) [reply client-generator/quasar.md]: Of course, could be nice, but I've tried to keep the same format as in Vue generator. I can modify both if you want.

File: client-generator/quasar.md (hunk 2 above)
Comment (devel-pa) [reply client-generator/quasar.md]: Thx, I'll keep in mind.

**Comment by dunglas** [comment]
Thanks @devel-pa

**Pull request merged**